py_mmd_tools/xml_util.py

```diff
@@ -27,11 +26,7 @@ def xml_check(xml_file):
             xml = ET.parse(xml_file)
             return True
         except XMLSyntaxError:
-            #try:
-            #    xml = ET.XML(bytes(bytearray(xml_file, encoding="utf-8")))
-            #    return True
-            #except XMLSyntaxError:
-            raise # return False
+            raise 
```

Comment:
which part is unnecessary?

Reply:
The method was here to be able to find the exact reason for the failure - in case the file is empty or doesn't exist ET will raise an OSError (which I found to bee too generic comp[ared to file not found / empty) - but yes, at this point, with the exceptions now handled by ET with OSError also in other places, it make sense if you want to remove it.

Reply:
aha - I understand... so you're only checking the syntax, right?

Reply:
I made a few changes and added a comment. I can't push to your repo, so I will close this PR and merge from a branch afterwards.